projects/clr/hipamd/src/hip_conversions.hpp

```diff
@@ -994,4 +994,12 @@ inline hipMemcpy3DParms getMemcpy3DParms(const hipMemcpy3DPeerParms& desc) {
   params.kind = hipMemcpyDeviceToDevice;
   return params;
 }
-};  // namespace hip
\ No newline at end of file
+
+inline hipArrayMemoryRequirements getArrayMemoryRequirements(
+    const hipArrayMemoryRequirements& desc) {
+  hipArrayMemoryRequirements requirements;
+  requirements.alignment = desc.alignment;
+  requirements.size = desc.size;
+  return requirements;
+}
```

Comment:
NIT: I think you can simplify the function down to.

```suggestion
inline hipArrayMemoryRequirements getArrayMemoryRequirements(
    const hipArrayMemoryRequirements& desc) {
  return hipArrayMemoryRequirements{desc.alignment, desc.size};
}
```